server/src/main/java/com/objectcomputing/checkins/security/OAuthCsrfFilter.java

```diff
@@ -0,0 +1,33 @@
+package com.objectcomputing.checkins.security;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.context.env.Environment;
+import io.micronaut.core.async.publisher.Publishers;
+import io.micronaut.http.*;
+import io.micronaut.http.annotation.Filter;
+import io.micronaut.http.cookie.Cookie;
+import io.micronaut.http.filter.OncePerRequestHttpServerFilter;
+import io.micronaut.http.filter.ServerFilterChain;
+import org.reactivestreams.Publisher;
+
+
+
+@Filter(value = {"/services/*", "/services/*"})
```

Comment:
Am I missing something here? aren't these two the same?

Reply:
fixed it.